From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?J=C3=B6rn=20Seidel?= <joern.seidel@casoon.de>
Date: Wed, 30 Sep 2026 20:35:46 +0200
Subject: Relief: Produktname, Bundle-ID und Profilverzeichnis
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Chromiums vorgesehener Weg für Produktnamen (BRANDING); IDS_PRODUCT_NAME
für Menüleiste und Über-Seite; CrProductDirName, damit Relief sein
eigenes Profil unter ~/Library/Application Support/Relief anlegt und neben
Chrome/Chromium läuft (Paket 36).

diff --git a/chrome/app/app-Info.plist b/chrome/app/app-Info.plist
index 6fada0a2b1a1f4b5675fc657a3a7b2337e43c4a1..1a8ac7ad701685f280ef86747f792fe71c0027bc 100644
--- a/chrome/app/app-Info.plist
+++ b/chrome/app/app-Info.plist
@@ -157,6 +157,8 @@
 	<string>6.0</string>
 	<key>CFBundleName</key>
 	<string>${CHROMIUM_SHORT_NAME}</string>
+	<key>CrProductDirName</key>
+	<string>Relief</string>
 	<key>CFBundlePackageType</key>
 	<string>APPL</string>
 	<key>CFBundleSignature</key>
diff --git a/chrome/app/chromium_strings.grd b/chrome/app/chromium_strings.grd
index d5f9a09fc06cb18a4a393ff29f94550dec70aaec..3a8dfe21f53fc5ff82e25c96aca790b980b0abc5 100644
--- a/chrome/app/chromium_strings.grd
+++ b/chrome/app/chromium_strings.grd
@@ -294,10 +294,10 @@ If you update this file, be sure also to update google_chrome_strings.grd. -->
         </then>
         <else>
           <message name="IDS_PRODUCT_NAME" desc="The Chrome application name" translateable="false">
-            Chromium
+            Relief
           </message>
           <message name="IDS_SHORT_PRODUCT_NAME" desc="The Chrome application short name." translateable="false">
-            Chromium
+            Relief
           </message>
         </else>
       </if>
diff --git a/chrome/app/theme/chromium/BRANDING b/chrome/app/theme/chromium/BRANDING
index f8363d5b294fe210ebfdd18374e95b54d6ff5c17..fcce26a78217774ee1afa2fd7de7428cc7b43ecf 100644
--- a/chrome/app/theme/chromium/BRANDING
+++ b/chrome/app/theme/chromium/BRANDING
@@ -1,10 +1,10 @@
 COMPANY_FULLNAME=The Chromium Authors
 COMPANY_SHORTNAME=The Chromium Authors
-PRODUCT_FULLNAME=Chromium
-PRODUCT_SHORTNAME=Chromium
-PRODUCT_INSTALLER_FULLNAME=Chromium Installer
-PRODUCT_INSTALLER_SHORTNAME=Chromium Installer
+PRODUCT_FULLNAME=Relief
+PRODUCT_SHORTNAME=Relief
+PRODUCT_INSTALLER_FULLNAME=Relief Installer
+PRODUCT_INSTALLER_SHORTNAME=Relief Installer
 COPYRIGHT=Copyright @LASTCHANGE_YEAR@ The Chromium Authors. All rights reserved.
-MAC_BUNDLE_ID=org.chromium.Chromium
+MAC_BUNDLE_ID=de.casoon.relief
 MAC_CREATOR_CODE=Cr24
 MAC_TEAM_ID=
